src/main/java/com/eternalcode/parcellockers/parcel/ParcelLockerRepositoryJdbcImpl.java

```diff
@@ -0,0 +1,91 @@
+package com.eternalcode.parcellockers.parcel;
+
+import com.eternalcode.parcellockers.database.JdbcConnectionProvider;
+import com.eternalcode.parcellockers.shared.Position;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.TimeUnit;
+
+public class ParcelLockerRepositoryJdbcImpl implements ParcelLockerRepository {
+
+    private final JdbcConnectionProvider provider;
+
+    public ParcelLockerRepositoryJdbcImpl(JdbcConnectionProvider provider) {
+        this.provider = provider;
+    }
+
+    @Override
+    public CompletableFuture<Void> save(ParcelLocker parcelLocker) {
+        return CompletableFuture.runAsync(() -> this.provider.executeUpdate("INSERT INTO `parcelLockers` (`uuid`, `description`, `position`, `size`) VALUES (" + parcelLocker.getUuid().toString() + ", " + parcelLocker.getDescription() + ", " + parcelLocker.getPosition().toString() + ")"))
```

Comment:
Zalecałbym użycia bindowania parametrów w zapytaniu SQL

Reply:
Podaj example

Reply:
PreparedStatement stmt=con.prepareStatement(query);

stmt.setInt(1,deptno); //1 is for the first question mark